test/e2e/workflow_test.go

```diff
@@ -198,16 +198,27 @@ var _ = Describe("Validate the persistence ", Ordered, func() {
 				}
 				Expect(h.Status).To(Equal(upStatus), "Pod health is not UP")
 				for _, c := range h.Checks {
-					if c.Name == "Database connections health check" {
-						Expect(c.Status).To(Equal(upStatus), "Pod's database connection is not UP")
-						return true
+					if withPersistence {
+						if c.Name == "Database connections health check" {
+							Expect(c.Status).To(Equal(upStatus), "Pod's database connection is not UP")
+							return true
+						}
+					} else {
+						Expect(c.Name).NotTo(Equal("Database connections health check"), "persistence configuration found in workflow")
 					}
 				}
+				if !withPersistence {
+					return true
+				}
 			}
 			return false
-		}, 12*time.Minute).Should(BeTrue())
+		}, 1).Should(BeTrue())
```

Comment:
It should not take more than 1 minute to stabilize the pod's health status. The previous value, 12 minutes, is too long for something that should take seconds. Worst case it would take 12 minutes for the test to fail and that I think it's too long.

Reply:
Note that 1 seems to be `1 second`. 
https://github.com/onsi/gomega/blob/32ef35ea6e233acc84502d9ec90c106aeec1615d/gomega_dsl.go#L245
I will add a `*time.Minute` back.